### PR TITLE
feat: add the ability to configure the block as scorable from Studio

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds the ability to configure the component of the XBlock as scorable from Studio.

### How To Test
1. Install this branch in your environment.
2. Add a block in a unit from Studio.
3. As an instructor, you should be able to select the component as scorable or no-scorable from the component settings in Studio.
4. As a student only the `save` button should be available.

#### Studio
image
#### LMS
image

### Limitations
The current implementation was based on [H5P XBlock](https://github.com/edly-io/h5pxblock), however, it has a limitation with the following use case:

1. As an instructor I define a component as `scorable=True`
2. As a student I make my submission.
3. As an instructor I grade the student's submission.
5. As a student I can see my progress in the course.
image
6. As an instructor I define the same component as `scorable=False`
7. As a student, I no longer get the component score, but it still counts in the course progress.
image

### CODE REVIEW

**Comment by Ian2012** [comment]
@BryanttV There is a command to recalculate the grades of a course, can you run that command and verify if the issue persists?

**Comment by BryanttV** [comment]
@Ian2012, Could you confirm me which is that command?

File: mindmap/mindmap.py [hunk 1]
```diff
@@ -29,13 +29,24 @@
...
 
-    has_score = True
-    icon_class = "problem"
+    has_score = Boolean(
+        display_name=_("Is scorable?"),
+        help=_(
+            "Whether the component is scorable. If is scorable, the student "
+            "can submit the mind map and receive a score from the instructor. "
+            "If it is not scorable, the student only can save the mind map. "
+            "Important: Changing from scorable to not scorable, the progress "
```
Comment (mariajgrimaldi) [inline mindmap/mindmap.py:43]:
```suggestion
            "WARNING: Changing from scorable to not scorable, the progress "
```

**Comment by Ian2012** [comment]
@BryanttV here is the documentation https://internal.docs.edunext.co/en/latest/internal/Teams/Service_Delivery/customer-service/manage_courses/course_grades.html?highlight=grade#compute-grades-in-a-frozen-course

File: mindmap/mindmap.py (hunk 1 above)
Comment (BryanttV) [reply mindmap/mindmap.py]: Done

**Review by mariajgrimaldi — COMMENTED** [review]
Working expected! Thanks

**Pull request merged**